en/Data Structures/Graph/Graph.md

```diff
@@ -0,0 +1,15 @@
+
+# Graph
+
+A Graph is a non-linear data structure consisting of vertices and edges. The vertices are sometimes also referred to as nodes and the edges are lines or arcs that connect any two nodes in the graph. More formally a Graph is composed of a set of vertices( V ) and a set of edges( E ). The graph is denoted by G(E, V).
+## Components of a Graph
+
+1. **Vertices**: Vertices are the fundamental units of the graph. Sometimes, vertices are also known as vertex or nodes. Every node/vertex can be labeled or unlabelled.
+
+2. **Edges**: Edges are drawn or used to connect two nodes of the graph. It can be ordered pair of nodes in a directed graph. Edges can connect any two nodes in any possible way. Sometimes, edges are also known as arcs.
```

Comment:
ordered pair (v1, v2) with v1, v2 in V; set `{v1, v2}` in an undirected graph (may be just `{v1}` if a node is connected to itself)